src/core/execute.c

```diff
@@ -463,6 +469,10 @@ int exec_spawn(
                         environ,
                         cg_unified() > 0 ? subcgroup_path : NULL,
                         &pidref);
+
+        if (unit->manager->original_ambient_set != 0 && MANAGER_IS_USER(unit->manager))
+                capability_ambient_set_apply(0, /* also_inherit= */ false);
```

Comment:
Please cast to (void) since the retval is discarded.

Reply:
DONE